### PR TITLE
fix: Improved Email authentication error messages

### PR DESCRIPTION
**Pull request by klkucaj**
Closes: #2051

## Pre-launch Checklist

- [x] I read the [Contribute](https://docs.serverpod.dev/contribute) page and followed the process outlined there for submitting PRs.
- [x] This update contains only one single feature or bug fix and nothing else. (If you are submitting multiple fixes, please make multiple PRs.)
- [x] I read and followed the [Dart Style Guide](https://dart.dev/guides/language/effective-dart/style) and formatted the code with [dart format](https://dart.dev/tools/dart-format).
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`), and made sure that the documentation follows the same style as other Serverpod documentation. I checked spelling and grammar.
- [ ] I added new tests to check the change I am making.
- [x] All existing and new tests are passing.
- [ ] Any breaking changes are documented below.

### CODE REVIEW

**Comment by klkucaj** [comment]
I have read the CLA Document and I hereby sign the CLA

File: modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart [hunk 1]
```diff
@@ -216,6 +216,10 @@ class Emails {
       where: (t) => t.userId.equals(userId),
     );
     if (auth == null) {
+      session.log(
+        'userId: \'$userId\' is invalid!',
```
Comment (Isakdl) [inline modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart:220]: Suggestion: Change the outer ' to ". Then you don't have to escape them inside the string.

**Review by Isakdl — CHANGES_REQUESTED** [review]
Looking good! :) 

Just a small change on the strings, see the comment.

File: modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart (hunk 1 above)
Comment (klkucaj) [reply modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart]: Done!

**Review by Isakdl — APPROVED** [review]
LGTM

**Pull request merged**